rosa/aws-load-balancer-operator/deploy-aws-lbo.sh

```diff
@@ -1,17 +1,21 @@
-#!/bin/sh
-oc create ns aws-load-balancer-operator
-set +e
+#!/bin/bash
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+echo "--> Fetch cluster information"
+
 export ROSA_CLUSTER_NAME=$(oc get infrastructure cluster -o=jsonpath="{.status.infrastructureName}"  | sed 's/-[a-z0-9]\{5\}$//')
 export REGION=$(oc get infrastructure cluster -o=jsonpath="{.status.platformStatus.aws.region}")
 export OIDC_ENDPOINT=$(oc get authentication.config.openshift.io cluster -o jsonpath='{.spec.serviceAccountIssuer}' | sed  's|^https://||')
 export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
 export CLUSTER_NAME=$(oc get infrastructure cluster -o=jsonpath="{.status.infrastructureName}")
+export AWS_DEFAULT_REGION=$REGION
```

Comment:
useful to set this in case the rosa cluster is not in what aws cli knows to be its region.